templates/html/.filters/all.js

```diff
@@ -30,6 +31,14 @@ module.exports = ({ Nunjucks, Markdown, OpenAPISampler }) => {
   Nunjucks.addFilter('isArray', (arr) => {
     return Array.isArray(arr);
   });
+  
+  Nunjucks.addFilter('isObject', (obj) => {
+    return typeof obj === 'object' && obj !== null;
+  });
+
+  Nunjucks.addFilter('prettyPrint', (obj) => {
+    return JSON.stringify(obj, null, 4);
+  });
```

Comment:
As pointed out below, you'll not need this.

Reply:
Beautiful. Thanks for that. It's a great change. I've updated all.js and schema-prop.html